test/cli.py

```diff
@@ -683,6 +683,46 @@ def test_013_cli_labels(self):
         state_files = [Path(p).stem for p in cache.get_all(cache_dir)]
         assert state_files == ["linear_d5b1df11_state", "linear2_80b93950_state"]
 
+    def test_014_report_on_failed_build(self):
+
+        # Run benchit on groq device (will fail since the HW is not available)
```

Comment:
This is a nit, you should probably ignore me, but: in a perfect world we *would* have Groq hardware available in CI. So this is kinda technical debt since eventually it will need to change.

Is there a cleaner way to fail a build? What about the methodology in the resume test? https://github.com/groq/mlagility/blob/8787a42209fd7799a3a8c9c7ecc2ddf610a6fef5/test/cli.py#L594

Reply:
Modified to ensure the test is indeed failing